WordPress/src/main/res/layout/delete_site_preference.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I am getting a lot of suggestions from AS about right to left languages, performance improvements and missing content description for the image view in this file. Not all of it is applicable, so I'll leave it to your judgement if there are changes you want to make.